fees/equilibria.ts

```diff
@@ -0,0 +1,57 @@
+import { FetchOptions, FetchV2, SimpleAdapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+
+const fetch: FetchV2 = async (option: FetchOptions) => {
+  const dailyFees = option.createBalances();
+  const dailyRevenue = option.createBalances();
+
+  const logs = await option.getLogs({
+    targets: [
```

Comment:
mostly good except for this part, would prefer creating a map of chains and rewardPools, and using only relevant contract here, you can get the chain key  with  `options.chain`, this way, we avoid unintentional side effects.